### PR TITLE
created addaptive slider

### PR DESCRIPTION
**Pull request by ghost**
files + foto + code

### CODE REVIEW

**Comment by ghost** [comment]
https://nolik2000.github.io/gci16.fossasia.org/

File: img/README.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+for slider
```
Comment (jigyasa-grover) [inline img/README.txt:1]: Is this file required ?

File: js/README.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+For slider
```
Comment (jigyasa-grover) [inline js/README.txt:1]: This can be skipped afaik :)

**Comment by jigyasa-grover** [comment]
@oanarosca @PolBaladas @m1guelpf @Abhi2424shek @rhemon @fossasia/gci16-maintainers  Could you pls help resolve this :)
@nolik2000 Feel free to follow up here or ask your queries in #gci channel on FOSSASIA Slack.

**Comment by ghost** [comment]
ok, thanks)))

**Comment by abishekvashok** [comment]
@jig08 i don't have right access as I am not a maintainer yet

**Comment by abishekvashok** [comment]
But I could help @nolik2000 so please ping me on slack @nolik2000

File: img/README.txt (hunk 1 above)
Comment (m1guelpf) [reply img/README.txt:1]: No

**Comment by niccokunzmann** [comment]
@nolik2000 I remember that you added this slider once. Good news: It became easier to add sliders. 
https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/slides.yml
is the file where you can add sliders to and then, you only need to add the pictures.
When I click the preview on your site, I do not see the slider.
To merge, would you like to resolve the conflicts?

**Comment by niccokunzmann** [comment]
What are your thoughts? How would you like to go on?

**Comment by jigyasa-grover** [comment]
With reference to [rules in CONTRIBUTING.md](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md), closing this.
